plugins/inventory/proxmox.py

```diff
@@ -443,7 +443,8 @@ def _get_vm_config(self, properties, node, vmid, vmtype, name):
     def _get_vm_status(self, properties, node, vmid, vmtype, name):
         ret = self._get_json("%s/api2/json/nodes/%s/%s/%s/status/current" % (self.proxmox_url, node, vmtype, vmid))
         properties[self._fact('status')] = ret['status']
-        properties[self._fact('qmpstatus')] = ret['qmpstatus']
+        if self.get_option('want_facts') and vmtype == 'qemu':
```

Comment:
This function is only called when `self.get_option('want_facts')`. Thus it doesn't make sense to check that here again.
```suggestion
        if vmtype == 'qemu':
```

Reply:
Oh, yeah, right... :sweat_smile: